cli/expvar.go

```diff
@@ -0,0 +1,53 @@
+package cli
+
+import (
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/yandex/pandora/core/engine"
+	"github.com/yandex/pandora/lib/monitoring"
+)
+
+func newEngineMetrics() engine.Metrics {
+	return engine.Metrics{
+		Request:        monitoring.NewCounter("engine_Requests"),
+		Response:       monitoring.NewCounter("engine_Responses"),
+		InstanceStart:  monitoring.NewCounter("engine_UsersStarted"),
+		InstanceFinish: monitoring.NewCounter("engine_UsersFinished"),
+	}
+}
+
+func startReport(m engine.Metrics) {
+	evReqPS := monitoring.NewCounter("engine_ReqPS")
+	evResPS := monitoring.NewCounter("engine_ResPS")
+	evActiveUsers := monitoring.NewCounter("engine_ActiveUsers")
+	evActiveRequests := monitoring.NewCounter("engine_ActiveRequests")
+	requests := m.Request.Get()
+	responses := m.Response.Get()
+	go func() {
+		var requestsNew, responsesNew int64
+		// TODO(skipor): there is no guarantee, that we will run exactly after 1 second.
+		// So, when we get 1 sec +-10ms, we getting 990-1010 calculate intervals and +-2% RPS in reports.
+		// Consider using rcrowley/go-metrics.Meter.
+		for range time.NewTicker(1 * time.Second).C {
```

Comment:
Floating point is ok+) we can even specify RPS in this way, for examle, 0.5 RPS means 1 request per 2 seconds.